src/main/java/com/kobylynskyi/graphql/codegen/mapper/GraphqlTypeToJavaTypeMapper.java

```diff
@@ -167,6 +167,36 @@ private static String wrapIntoJavaCollection(String type) {
         return String.format("Collection<%s>", type);
     }
 
+    /**
+     * Wrap java type into {@link java.util.concurrent.CompletableFuture}. E.g.: "String" becomes "CompletableFuture<String>"
+     *
+     * @param type Anything that will be wrapped into CompletableFuture<>
+     * @return String wrapped into CompletableFuture<>
+     */
+    private static String wrapIntoJavaCompletableFuture(String type) {
+        return String.format("CompletableFuture<%s>", type);
```

Comment:
@ReneBL please use the fully-qualified name `java.util.concurrent.CompletableFuture` because the generated class will have problems with imports.

Reply:
I've added the new method _MapperUtils#getImportsForFieldDefinition_ in order to address this issue.